test/integration_test/common_test.go

```diff
@@ -98,30 +102,77 @@ func setup(t *testing.T) {
 	require.NoError(t, err, "Error initializing volume driver %v", volumeDriverName)
 }
 
+func SetupStork(t *testing.T) {
+	doOnce.Do(func() {
+		if passed := t.Run("setup", setup); !passed {
+			t.FailNow()
+		}
+	})
+}
+
+func TestMock1(t *testing.T) {
+	SetupStork(t)
+	t.Run("Mocktest-1", testMock)
+}
+
+func TestMock2(t *testing.T) {
+	SetupStork(t)
+	t.Run("Mocktest-2", testMock)
+}
+
+func TestMock3(t *testing.T) {
+	SetupStork(t)
+	t.Run("Migration", testMigration)
+}
+
 func TestMain(t *testing.T) {
```

Comment:
TestMain is no longer required.

Can the setup be done from init() instead? Don't need to change this function then, just need to remove the call to setup from here

Reply:
Moved setup to init() function.